xla/service/gpu/nccl_types.h

```diff
@@ -17,7 +17,17 @@ limitations under the License.
 #define XLA_SERVICE_GPU_NCCL_TYPES_H_
 
 #if XLA_ENABLE_XCCL
+// Common place for all collective thunks to include nccl/rccl headers.
+#if TENSORFLOW_USE_ROCM
+#include "rocm/rocm_config.h"
+#if (TF_ROCM_VERSION >= 50200)
+#include "rocm/include/rccl/rccl.h"
+#else
+#include "rocm/include/rccl.h"
+#endif
+#else
 #include "third_party/nccl/nccl.h"
```

Comment:
Can you please add this comment:

```
// IWYU pragma: export
```

So that our include cleaner tool allows to include nccl_types.h instead of nccl.h?

Reply:
sure, did it !

Reply:
Thank you :)